pythonforandroid/distribution.py

```diff
@@ -129,6 +128,7 @@ def get_distribution(cls, ctx, name=None, recipes=[],
         # If there was a name match but we didn't already choose it,
         # then the existing dist is incompatible with the requested
         # configuration and the build cannot continue
+<<<<<<< HEAD
```

Comment:
Looks like something went wrong during the rebasing :confused: 
Apparently I have write access to the branch so I'll pick this up

Reply:
Fixed